help/destinations/catalog/crm/salesforce.md

```diff
@@ -0,0 +1,161 @@
+
+* Within the [!DNL Experience Platform], navigate to **Destinations**. 
+* Select the **Catalog** tab and search for *Salesforce CRM*, or select *CRM* within **Categories**. 
+* Then select **[!UICONTROL Set up]**. After you have established a connection to the destination, the UI label changes to **[!UICONTROL Activate Segments]**.
+![Catalog](../../assets/catalog/crm/salesforce/catalog.png)
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, fill in the required fields and select **[!UICONTROL Connect to destination]**.
+
+1. Next, you are shown a wizard which assists you in configuring the Salesforce CRM destination. Create the new destination by selecting **[!UICONTROL Configure new destination]** and fill in the details below and select **[!UICONTROL Connect to destination]**.
+![Account Info](../../assets/catalog/crm/salesforce/account-info.png)
+
+*  **[!UICONTROL Password]**: Your Salesforce account password.
+*  **[!UICONTROL Client ID]**: Consumer Key generated during the [prerequisites](#prerequisites) step.
+*  **[!UICONTROL Client Secret]**: Consumer Secret generated during the [prerequisites](#prerequisites) step.
+*  **[!UICONTROL Username]**: Your Salesforce account username.
+
+1. If the details provided are valid, the UI displays a **Connected** status with a green check mark, you can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+![Destination Details](../../assets/catalog/crm/salesforce/destination-details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Custom Domain]**: Your Salesforce domain as shown within [prerequisites](#prerequisites).
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+The list of attribute mappings that can be set up for the [Salesforce REST API](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/dome_composite_upsert_example.htm?q=contacts) is given below.
+
```

Comment:
This table of identities should be moved to the Supported identities section. Please update that table according to what Salesforce supports, as mentioned in the template: https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/self-service-template.html?lang=en#supported-identities

Reply:
These are not linked to the *Supported identities*, its for [Map attributes and identities](https://experienceleague.adobe.com/docs/experience-platform/destinations/ui/activate/activate-segment-streaming-destinations.html?lang=en#mapping)

Reply:
[Vlad Falon](https://app.slack.com/team/U03BEN7G3RA)  [7:58 PM](https://launch-developers.slack.com/archives/D03G7C72HJ5/p1653402533002859)
[@Mohan Hariharan](https://launch-developers.slack.com/team/U030SRCTJEM) regarding the last comment, those are the supported identities. Supported identities describes what type of identifiers you can map in that step. Whatever Salesforce supports needs to go in the supported identities table.
 
[7:59](https://launch-developers.slack.com/archives/D03G7C72HJ5/p1653402552982979)
the activation section should be the same for all destinations, just like the template shows